daml-foundations/daml-ghc/daml-compiler/src/DA/Service/Daml/Compiler/Impl/Scenario.hs

```diff
@@ -10,26 +10,31 @@ module DA.Service.Daml.Compiler.Impl.Scenario (
   , EnableScenarioService(..)
   , withScenarioService
   , withScenarioService'
+  , SS.ScenarioServiceConfig
+  , SS.readScenarioServiceConfig
   ) where
 
 import DA.Daml.GHC.Compiler.Options
 import qualified DA.Daml.LF.ScenarioServiceClient as SS
 import qualified DA.Service.Logger                          as Logger
 import           Control.Monad.IO.Class                     (liftIO)
 import qualified Data.Text                                  as T
+import System.IO
 
 withScenarioService
     :: Logger.Handle IO
+    -> SS.ScenarioServiceConfig
     -> (SS.Handle -> IO a)
     -> IO a
-withScenarioService loggerH f = do
+withScenarioService loggerH conf f = do
+    hPrint stderr conf
```

Comment:
This seems surprising? If we want to print it, at least we should say what we are printing. Or are these debug droppings?

Reply:
oops debug output